### DISCUSSION TITLE
Using InfoNCE Loss to Compare CEBRA-Behavior vs. CEBRA-Time models

### DISCUSSION THREAD

**Discussion by rhingo** [AdaptiveMotorControlLab/CEBRA#159] [General]
Hi, thanks for the great paper! I had a quick question about the interpretation of the InfoNCE loss as a goodness of fit metric. I understand that the paper discusses how the InfoNCE can be used to compare models with different auxiliary behavioral variables as the supervisory signal (in the hypothesis-driven mode). However, can I also compare the InfoNCE loss between a CEBRA-Behavior and CEBRA-Time models? Specifically, I am interested in answering the question, does the behavior auxiliary variable improve the embedding compare to a time-only model. Thanks!

**Comment by MMathisLab** [comment]
Thanks! Yes, if same parameters otherwise and comparing at the same snapshot in time, that is possible.
